test/extensions/filters/network/thrift_proxy/driver/BUILD

```diff
@@ -21,9 +21,7 @@ py_binary(
     python_version = "PY2",
```

Comment:
can we remove this (and i guess the six dep below) ?

Reply:
Yes.